wire-runtime/src/main/java/com/squareup/wire/Message.java

```diff
@@ -102,7 +102,7 @@ public final void encode(OutputStream stream) throws IOException {
    * Superclass for protocol buffer message builders.
    */
   public abstract static class Builder<T extends Message<T, B>, B extends Builder<T, B>> {
-    // Lazily-instantiated buffer and writer of this message's unknown fields.
+    transient ByteString unknownFieldsByteString = ByteString.EMPTY;
```

Comment:
Not sure why these are `transient`, it seems that at least `unknownFieldsBuffer` is part of the object state. But I can't really imagine anyone using Java serialization on this class.